runtime/compiler/env/VMJ9.h

```diff
@@ -895,6 +895,8 @@ class TR_J9VMBase : public TR_FrontEnd
     */
    TR::KnownObjectTable::Index delegatingMethodHandleTarget(
       TR::Compilation *comp, TR::KnownObjectTable::Index dmhIndex, bool trace);
+   virtual TR::KnownObjectTable::Index delegatingMethodHandleTarget(
```

Comment:
Somehow I am not very fond of using the same name because it can be confusing. This new method is just one small part of the original `delegatingMethodHandleTarget`. Maybe add "Helper" at the end.